docs/general/wallets.md

```diff
@@ -0,0 +1,155 @@
+important to keep the private key secure.
+
+:::tip Not your keys, not your tokens!
+
+With custodial wallets (like accounts central exchanges), another party controls your private keys.
+With non-custodial wallets, only you have access to your account's private key.
+
+:::
+
+To realize the multichain vision of Polkadot, it is important to have non-custodial wallets that
+make it convenient to interact with multiple blockchains in Polkadot ecosystem.
+
+## Non-Custodial Wallets
+
+:::caution
+
+You should use your own due diligence in researching and using the wallets listed below. For any
+issues related to these wallet, reach out to their support teams directly.
+
+:::
+
+Typically, the account keys are either stored and accessed through a browser extension or a
+smartphone app (which are considered as a hot wallets as they are exposed online) or an airgapped
+device or a hardware wallet (which considered as a cold wallets as they are offline).
+
+### Browser Wallets
+
+It is possible to use your browser to interact with your blockchain account through the wallets
+listed below. Typically, these wallets use the browser extension as a keystore for the blockchain
+accounts. Some of the browser extensions also allow interacting with airgapped devices and hardware
+wallets in a secure way.
+
+| Wallet                                | Browsers                                    | Staking and Nomination Pools | NFTs    | Crowdloans | Ledger support | Governance | Other features                   |
+| ------------------------------------- | ------------------------------------------- | ---------------------------- | ------- | ---------- | -------------- | ---------- | -------------------------------- |
+| [Enkrypt](https://www.enkrypt.com/)   | Brave, Chrome, Edge, Firefox, Opera, Safari | No, No                       | **Yes** | No         | **Yes**        | No         | [Enkrypt Features](#enkrypt)     |
+| [PolkaGate](https://polkadotjs.plus/) | Firefox                                     | **Yes**, **Yes**             | No      | **Yes**    | **Yes**        | No         | [PolkaGate features](#polkagate) |
+| [SubWallet](https://subwallet.app/)   | Brave, Chrome, Edge, Firefox                | **Yes**, **Yes**             | **Yes** | **Yes**    | **Yes**        | No         | [SubWallet features](#subwallet) |
+| [Talisman](https://www.talisman.xyz/) | Brave, Chrome, Edge, Firefox                | **Yes**, **Yes**             | **Yes** | **Yes**    | **Yes**        | No         | [Talisman features](#talisman)   |
+
+### Mobile Wallets
+
+Mobile wallets are fully packaged apps with the key-store as well as functionalities unlike browser
+based wallets, which typically use a browser extension as a key-store (some wallets put a lot more
+functionality into the extensions) and the browser for interacting with the blockchain networks.
+Some mobile wallets also support hardware wallets which provide bluetooth connectivity.
```

Comment:
"Mobile wallets are fully packaged apps that allow all the featured actions, as well as the storage and management of your accounts, through the mobile app. Contrary to browser extensions, mobile wallets usually can't connect to third-party web apps. Also, some mobile wallets provide support for hardware wallets with bluetooth connectivity"

Reply:
Added the suggestion